### PR TITLE
Bolt nflow-explorer-ng to nflow build and make it the default UI

### PR DESCRIPTION
**Pull request by eputtone**
Demo deployed here: https://bank.nflow.io/nflow-explorer-ng/
Old Explorer for reference: https://bank.nflow.io/nflow/ui/explorer/

- bolted nflow-explorer-ng into the main build, and package it by default in nflow-jetty and nflow-tests
- Updated README.md
- Removed redundant auto-generated files from the UI template
- Upgraded to latest minor- and patch version plus latest react-scripts
- Smaller padding around the content area
- Fixed all found browser console errors
- Renamed "Statistics summary" to "Instance statistics" and made it more readable (workflow definitions)
- Moved "update state" from manage-tab to below state table (workflow instance)

### CODE REVIEW

File: nflow-explorer-ng/src/App.scss [hunk 1]
```diff
@@ -2,5 +2,5 @@
   margin-left: auto;
   margin-right: auto;
   max-width: 1920px;
-  padding: var(--size-xl);
+  padding: var(--size-m);
```
Comment (jsyrjala) [inline nflow-explorer-ng/src/App.scss:5]: I think this would look visually better:
```
padding: 0;
```

File: nflow-explorer-ng/package.json [hunk 2]
```diff
@@ -1,47 +1,47 @@
...
     "husky": "^4.3.8",
-    "lint-staged": "^11.0.0",
+    "lint-staged": "^11.2.6",
     "lodash": "^4.17.21",
-    "moment": "^2.29.1",
-    "mui-datatables": "^3.8.2",
+    "moment": "^2.29.3",
+    "mui-datatables": "^3.8.5",
     "mui-rff": "^3.0.16",
     "node-sass": "^6.0.1",
```
Comment (jsyrjala) [inline nflow-explorer-ng/package.json:36]: `node-sass` is deprecated: https://www.npmjs.com/package/node-sass. Also `node-sass` doesn't build on Apple M1.

`sass` seems be drop-in replacement:

```
    "sass": "^1.52.2"
```

File: nflow-explorer-ng/src/App.scss (hunk 1 above)
Comment (jsyrjala) [reply nflow-explorer-ng/src/App.scss:5]: Or not to have padding line at all, same effect.

Comment (eputtone) [reply nflow-explorer-ng/src/App.scss]: It's in now. Though you're also welcome to make your own PRs for random pixel tinkerings ;)

File: nflow-explorer-ng/package.json (hunk 2 above)
Comment (eputtone) [reply nflow-explorer-ng/package.json]: This is also done now :+1:

**Review by jsyrjala — APPROVED** [review]

**Pull request merged**